poc_iot_verifier/src/poc.rs

```diff
@@ -1,13 +1,15 @@
 use crate::{
-    entropy::ENTROPY_LIFESPAN, gateway_cache::GatewayCache, last_beacon::LastBeacon, Error, Result,
+    entropy::ENTROPY_LIFESPAN,
```

Comment:
i have heavily refactored this module here: https://github.com/helium/oracles/pull/251

I'll sort out the conflicts and carry across the anyhow changes after #252 gets merged